desktop/core/src/desktop/auth/api_authentications_tests.py

```diff
@@ -47,6 +48,10 @@ def setUp(self):
       }
     )
 
+  @classmethod
+  def setUpClass(cls):
+    if not AUTH.JWT.IS_ENABLED.get():
+      raise SkipTest
```

Comment:
One issue now is that the tests won't run forever more or less as it is False by default.

Does it work if you set the value of DEFAULT_AUTHENTICATION_CLASSES with the `JwtAuthentication` class to force turn it on? (note: sometimes it might not be enough when interacting with settings.py properties are those get loaded at startup)

Reply:
Agreed, so shall we remove it then? Either way the unit tests are isolated and doesn't depend whether the JwtAuthentication class is added or not.
> One issue now is that the tests won't run forever more or less as it is False by default.

Not sure what is meant with 'force turn it on', if the `is_enabled` flag is true, then the JwtAuthencation class will be added in DEFAULT_AUTHENTICATION_CLASSES and if anyone runs these unit tests, they will be executed. So both the unit tests and adding of JwtAuthentication class in settings.py depends on `is_enabled` @romainr .
> Does it work if you set the value of DEFAULT_AUTHENTICATION_CLASSES with the `JwtAuthentication` class to force turn it on? (note: sometimes it might not be enough when interacting with settings.py properties are those get loaded at startup)

Reply:
Yes, please no SkipTest, see if we can turn it on in setUpClass, then turn it off in tearDownClass